weblate/trans/tasks.py

```diff
@@ -228,8 +227,12 @@ def cleanup_suggestions():
 def update_remotes():
     """Update all remote branches (without attempt to merge)."""
     non_linked = Component.objects.with_repo()
-    for component in non_linked.iterator():
-        perform_update.delay("Component", component.pk, auto=True)
+
+    if settings.AUTO_UPDATE not in ("full", "remote", True, False):
+        return
+    else:
```

Comment:
The else is no longer needed here as there is return in the if clause, see also https://deepsource.io/gh/WeblateOrg/weblate/run/3b96cf45-4966-4da0-a4ef-29440c86121d/python/PYL-R1705